.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       uses: actions/setup-java@v4
       with:
         distribution: corretto
-        java-version: '17'
+        java-version: '11'
```

Comment:
This was on purpose using Java17 because it's faster than Java11

Reply:
I'll change the wording in setting key then because it's a bit confusing

Reply:
Fixed